### PR TITLE
Validate method parameters for template extension

### PR DESCRIPTION
**Pull request by angelozerr**
Validate method parameters for template extension

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java [hunk 1]
```diff
@@ -49,6 +49,15 @@ public void setResolvedType(ResolvedJavaTypeInfo resolvedType) {
 	public String getSourceType() {
 		return resolvedType != null ? resolvedType.getSignature() : null;
 	}
+	
+	/**
+	 * Returns the java source ctype and null otherwise.
```
Comment (fbricon) [inline qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java:54]: s/ctype/type

File: qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java [hunk 2]
```diff
@@ -49,6 +49,15 @@ public void setResolvedType(ResolvedJavaTypeInfo resolvedType) {
 	public String getSourceType() {
 		return resolvedType != null ? resolvedType.getSignature() : null;
 	}
+	
+	/**
+	 * Returns the java source ctype and null otherwise.
```
Comment (fbricon) [inline qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java:54]: s/ctype/type

Comment (angelozerr) [reply qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java]: fixed

File: qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java (hunk 1 above)
Comment (angelozerr) [reply qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/JavaMemberInfo.java]: fixed

**Comment by fbricon** [comment]
Thanks @angelozerr

**Pull request merged**